### PR TITLE
Add table

### PR DESCRIPTION
**Pull request by monzter50**
Create component pricing, I kept the basic design as said the task

### CODE REVIEW

File: src/components/Pricings.vue [hunk 1]
```diff
@@ -1,36 +1,69 @@
...
+          <div class="table-price select">
+              <div class="table-price__head">
+                  <h2 class="table-price__title">Premium</h2>
+                  <p class="table-price__price"> $1,299</p>
+              </div>
+              <div class="table-price__body">
+                  <ul class="table-price__list">
+                      <li class="table-price__items">Lorem ipsum</li>
+                      <li class="table-price__items">Lorem ipsum</li>
+                      <li class="table-price__items">Lorem ipsum</li>
```
Comment (henryraygan) [inline src/components/Pricings.vue]: add data for use cases real, _lorem ipsum_ is ambiguous

File: src/components/Pricings.vue [hunk 2]
```diff
@@ -1,36 +1,69 @@
...
+                  </ul>
+              </div>
+              <div class="table-price__select">
+            
+                  <button class="button-lg--outline-secondary">Select</button>
+                  <p class="table-price__small-title">lorem ipsum</p>
```
Comment (henryraygan) [inline src/components/Pricings.vue]: what is objetive this element on component?

File: src/assets/scss/components/_pricing.scss [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        list-style: none;
+        padding: 0;
+        margin: 0;
+    }
+    &__items{
+        display: block;
+        padding: .75rem 1rem;
+        margin-bottom: -1px;
+    }
+    &.select{
```
Comment (henryraygan) [inline src/assets/scss/components/_pricing.scss]: this class in another cases is good,  but template use BEM for naming clases please naming with a modifier. 

beacause class _"select"_ will posiblement confused with anothers components, BEM prevent this bad practice.

File: src/components/Pricings.vue [hunk 4]
```diff
@@ -1,36 +1,69 @@
...
+              </div>
+          </div>
+          <div class="table-price select">
+              <div class="table-price__head">
+                  <h2 class="table-price__title">Premium</h2>
+                  <p class="table-price__price"> $1,299</p>
```
Comment (henryraygan) [inline src/components/Pricings.vue]: Center text, please

File: src/assets/scss/components/_pricing.scss [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+}
+.table-price{
+    
+    display: flex;
+    flex-direction: column;
+    border-radius: .4em;
+    box-sizing: content-box;
+    // min-height: 400px;
+    height: 500px;
+    box-shadow: 0 4px 6px rgba(0, 0, 0, 0.5);
```
Comment (henryraygan) [inline src/assets/scss/components/_pricing.scss]: el color del box-shadow es muy fuerte, no tiene un buen contraste con el ui presente en el template

**Comment by henryraygan** [comment]
@monzter50 checkout this example for UI 

It is a suggestion, the icons are not necessary, but the distribution information and the selected colors create a good contrast with the current design context, and it is simple

pricing_ui_up

**Comment by monzter50** [comment]
*The data you should give me the real data since I use the lorem
Because is dummie data

*You can be most specifyc I dont understand I don't understand what
you mean by the component

*I combine metodology OOCSS with BEM Because I see OOCSS with BEM and
not specifyc what metedology use

*Perfect I correct the centered text and the shadow box

**Comment by henryraygan** [comment]
@monzter50 Can you answer the thread of each comment to have an individual discussion for each comment?

File: src/assets/scss/components/_pricing.scss [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+        }
+    }
+}
+.table-price{
+    
+    display: flex;
+    flex-direction: column;
+    border-radius: .4em;
+    box-sizing: content-box;
+    // min-height: 400px;
```
[the remark this answers is not in the document]
Comment (monzter50) [reply src/assets/scss/components/_pricing.scss]: Ok, i check this. 
Thanks 😃

[the remark this answers is not in the document]
Comment (monzter50) [reply src/assets/scss/components/_pricing.scss]: Captura

**Comment by monzter50** [comment]
I finished fix the comments their put. 
I wait for your comments.
Captura

**Comment by henryraygan** [comment]
#4

**Comment by henryraygan** [comment]
This pull request resolve issue: #4

**Pull request merged**